libs/dev-tools/src/lib/services/dr/create-dr-input.ts

```diff
@@ -10,6 +12,19 @@ const DEFAULT_TALLY_GAS_LIMIT = 50_000_000_000_000;
 const DEFAULT_GAS_PRICE = 2_000n;
 const DEFAULT_MEMO = new Uint8Array([]);
 
+const defaultDrConfig: DrConfig = {
+	commit_timeout_in_blocks: 50,
+	reveal_timeout_in_blocks: 5,
+	backup_delay_in_blocks: 2,
```

Comment:
these fields are not used, right?
I suppose you added them to match the dr config on the core contracts

Reply:
Yeah, I'm ok with leaving them out too :)

Reply:
both are okay for me... maybe one thing less to "keep updated" 🤣